<?xml version="1.0"?>

<launch>

	<!-- Launches the AR.Drone driver -->
	<node name="ardrone_driver" pkg="ardrone_autonomy" type="ardrone_driver" output="screen" clear_params="true">
	    <param name="outdoor" value="0" /> <!-- If we are flying outdoors, will select more aggressive default settings -->
	    <param name="flight_without_shell" value="0" /> <!-- Changes internal controller gains if we are flying without the propeller guard -->
	    
	    <param name="altitude_max" value="3000" /> <!-- in millimeters = 3 meters = 9' -->
        <param name="altitude_min" value="50" /> <!-- in millimeters = 5cm = 2" -->
        <param name="euler_angle_max" value="0.1" /> <!-- maximum allowable body angle in radians = 5 degrees -->
        <param name="control_vz_max" value="200" /> <!-- maximum z velocity in mm/sec = 0.2m/sec -->
	    <param name="control_yaw" value="0.7" /> <!-- maximum rotation rate in radians/sec = 40 degrees per second (1/9 rev/sec) -->
	</node>
	
	<!-- Launches the keyboard controller -->
	  <node name="KeyboardController" pkg="ardrone_lab" type="keyboard_controller.py" required="true"/> --> 

	<!-- Launches the video controller -->
	<node name="DroneMaster" pkg="ardrone_lab" type="drone_master.py" required="true"/> -->

        <!-- Launches flight statistics display -->
	<node name="FlightstatsDisplay" pkg="ardrone_lab" type="flightstats_display.py" required="true"/> -->
        
        <!-- Launches PID calibrater GUI -->
	<node name="PID_Calibrater" pkg="ardrone_lab" type="pid_calibrater.py" required="True"/> -->

</launch>
